spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -67,6 +67,8 @@
           flow_path: 'standard',
           irs_reproofing: false,
           step: 'verify',
+          same_address_as_id: true,
+          pii_like_keypaths: [[:same_address_as_id], [:state_id, :state_id_jurisdiction]],
```

Comment:
I just checked, and the first thing Analytics#track_events does is `attributes.delete(:pii_like_keypaths)`, so that's why they don't show up in the feature specs.